src/rpdk/core/contract/suite/handler_update.py

```diff
@@ -32,30 +32,6 @@ def updated_resource(resource_client):
         )
 
 
-@pytest.mark.update
-@pytest.mark.read
-def contract_update_read_success(updated_resource, resource_client):
-    # should be able to use the created model
-    # to read since physical resource id is immutable
-    _create_request, _created_model, _update_request, updated_model = updated_resource
-    assert resource_client.is_primary_identifier_equal(
-        resource_client.primary_identifier_paths, _created_model, updated_model
-    )
-    test_read_success(resource_client, updated_model)
-
-
-@pytest.mark.update
-@pytest.mark.list
-def contract_update_list_success(updated_resource, resource_client):
-    # should be able to use the created model
-    # to read since physical resource id is immutable
-    _create_request, _created_model, _update_request, updated_model = updated_resource
-    assert resource_client.is_primary_identifier_equal(
-        resource_client.primary_identifier_paths, _created_model, updated_model
-    )
-    assert test_model_in_list(resource_client, updated_model)
-
-
 @pytest.mark.update
 def contract_update_create_only_property(resource_client):
```

Comment:
I think a better solution would be to simply use the fixture in this test. that way we won't run into the issue of attempting to create the same resource twice because of overrides

Reply:
I like it. I will try to do this tomorrow. Furthermore, it's almost like on ANY test failure, I want to have cleanup attempted and re-init resources OR we should stop test exec on first error. Don't know what the feeling is on this.